quarkus-cli/src/test/java/io/quarkus/ts/quarkus/cli/QuarkusCliExtensionsIT.java

```diff
@@ -72,7 +71,7 @@ public void shouldListExtensionsUsingFull() {
     public void shouldListExtensionsUsingOtherPlatformVersions() {
         for (String expectedVersion : EXPECTED_PLATFORM_VERSIONS) {
             whenGetListExtensions("--origins", "--platform-bom=io.quarkus:quarkus-bom:" + expectedVersion);
-            assertListOriginsOptionOutput(expectedVersion);
+            assertListOriginsOptionOutput();
```

Comment:
I'm relaxing this assertion as for 999-SNAPSHOT, it's returning 2.0.2.Final dependencies which is correct, but I don't want to hardcode 2.0.2.Final here.